undefined: syscall.Kill

```
E:\GoProjects\tendermint\src\github.com\tendermint>set GOPATH=E:\GoProjects\test

E:\GoProjects\tendermint\src\github.com\tendermint>go get -u github.com/tendermint/abci/cmd/abci-cli
# github.com/tendermint/tmlibs/common
..\..\..\..\test\src\github.com\tendermint\tmlibs\common\os.go:56:9: undefined: syscall.Kill

E:\GoProjects\tendermint\src\github.com\tendermint>go env
set GOARCH=amd64
set GOBIN=
set GOEXE=.exe
set GOHOSTARCH=amd64
set GOHOSTOS=windows
set GOOS=windows
set GOPATH=E:\GoProjects\test
set GORACE=
set GOROOT=D:\Go
set GOTOOLDIR=D:\Go\pkg\tool\windows_amd64
set GCCGO=gccgo
set CC=gcc
set GOGCCFLAGS=-m64 -mthreads -fmessage-length=0
set CXX=g++
set CGO_ENABLED=1
set CGO_CFLAGS=-g -O2
set CGO_CPPFLAGS=
set CGO_CXXFLAGS=-g -O2
set CGO_FFLAGS=-g -O2
set CGO_LDFLAGS=-g -O2
set PKG_CONFIG=pkg-config
```

--- a/common/os.go
+++ b/common/os.go
@@ -35,6 +35,8 @@ func GoPath() string {
 	return path
 }
 
+// TrapSignal catches the SIGTERM and executes cb function. After that it exits
+// with code 1.
 func TrapSignal(cb func()) {
 	c := make(chan os.Signal, 1)
 	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
@@ -50,10 +52,13 @@ func TrapSignal(cb func()) {
 	select {}
 }
 
-// Kill the running process by sending itself SIGTERM
+// Kill the running process by sending itself SIGTERM.
 func Kill() error {
-	pid := os.Getpid()
-	return syscall.Kill(pid, syscall.SIGTERM)
+	p, err := os.FindProcess(os.Getpid())
+	if err != nil {
+		return err
+	}
+	return p.Signal(syscall.SIGTERM)
 }
 
 func Exit(s string) {

